docs/apache-airflow-providers-amazon/connections/aws.rst

```diff
@@ -32,19 +45,26 @@ Alternatively, one can pass credentials in as a Connection initialisation parame
 To use IAM instance profile, create an "empty" connection (i.e. one with no AWS Access Key ID or AWS Secret Access Key
 specified, or ``aws://``).
 
-
 Default Connection IDs
 -----------------------
 
 The default connection ID is ``aws_default``. If the environment/machine where you are running Airflow has the
 file credentials in ``/home/.aws/``, and the default connection has user and pass fields empty, it will take
 automatically the credentials from there.
 
-.. note:: Previously, the ``aws_default`` connection had the "extras" field set to ``{"region_name": "us-east-1"}``
+.. important:: Previously, the ``aws_default`` connection had the "extras" field set to ``{"region_name": "us-east-1"}``
     on install. This means that by default the ``aws_default`` connection used the ``us-east-1`` region.
     This is no longer the case and the region needs to be set manually, either in the connection screens in Airflow,
     or via the ``AWS_DEFAULT_REGION`` environment variable.
 
+.. caution:: If you do not set ``[database] load_default_connections`` to ``True``
+    most probably you do not have ``aws_default``. By historical reason Amazon Provider
+    components (Hooks, Operators, Sensors, etc.) fallback to default boto3 credentials strategy
+    in case of missing Connection ID, this behaviour is deprecated and will be removed in a future releases.
```

Comment:
```suggestion
    components (Hooks, Operators, Sensors, etc.) fallback to the default boto3 credentials strategy
    in case of a missing Connection ID. This behaviour is deprecated and will be removed in a future releases.
```